### ISSUE TITLE
kim_mouse atlas plots only one hemisphere in brainrender

### ISSUE THREAD

**Issue by alowet** [brainglobe/bg-atlasapi#114]
Using kim_mouse_50um, for which scaling issues have been solved (see #94), individual brain regions plot in only one hemisphere or the other. Consider this snippet:

```
from brainrender import Scene

atlas = 'kim_mouse_50um'

# Create a brainrender scene
scene = Scene(screenshots_folder='.', inset=False, atlas_name=atlas)

# add actors
acbc = scene.add_brain_region('Acb', alpha=0.3, color='blue', hemisphere='both')

# render
scene.render(interactive=False, zoom=1)
scene.screenshot(name='example_{}'.format(atlas))
```
This produces the following output:
![example_kim_mouse_50um](https://user-images.githubusercontent.com/33499753/157293857-5507fae9-c9ca-4b1d-ad42-060080b025a7.png)

As you can see, the nucleus accumbens gets plotted only in the left hemisphere. This is contrary to the behavior of `allen_mouse_25um` and the documented behavior of the `hemisphere` parameter. As far as I can tell, this behavior does *not* affect the `structure_from_coords` function, i.e. that function correctly returns the same region regardless of whether the point is in the left or the right hemisphere. It's currently unclear to me whether this is a problem with the atlas API itself or with brainrender, but I figured I'd report the bug here because of the difference between `allen_mouse_25um` and `kim_mouse_50um`. If I should go to the brainrender Github instead, let me know.

**Comment by adamltyson** [comment]
Hey @alowet, I'm not sure where the error lies, maybe @FedeClaudi has some insight?

Visualising the Acb outside of brainrender (`.brainglobe/kim_mouse_50um_v1.0/meshes/56.obj`) seems to show both hemispheres.

![Screenshot from 2022-03-09 11-31-51](https://user-images.githubusercontent.com/13147259/157433655-24ba6e12-b0d0-4745-8ce0-e5bd321ac606.png)

**Comment by IgorTatarnikov** [comment]
It seems there are some issues with the obj files for the Kim atlas. I sampled a few other random areas and there's a high chance that the obj files only contain one hemisphere.

Local file downloaded by bg-atlasapi:
![Kim_100_ACB_Local](https://github.com/brainglobe/bg-atlasapi/assets/61896994/b340e42f-eb1a-4f15-ad42-ed16dec0a3d1)

File downloaded directly from GIN:
![Kim_100_ACB_GIN](https://github.com/brainglobe/bg-atlasapi/assets/61896994/4d0c360a-6ee8-4ac7-8692-7abd226242e3)

Local copy for the same region in Allen 100 um:
![Allen_100_ACB](https://github.com/brainglobe/bg-atlasapi/assets/61896994/2b7a20a0-8337-4590-9eaf-e4d8da69c5e2)

**Comment by alessandrofelder** [comment]
![image](https://github.com/brainglobe/bg-atlasapi/assets/10500965/74a255eb-1d6f-46d3-81ea-50900d743f62)

I can reproduce in `brainrender-napari` and in Paraview. Is it possible that this affects only mesh files that are supposed to consist of several disconnected parts?

Maybe this issue needs to be transferred to `bg-atlasgen` as it's likely a problem with https://github.com/brainglobe/bg-atlasgen/blob/main/bg_atlasgen/atlas_scripts/kim_mouse.py? @IgorTatarnikov what do you think?

**Comment by IgorTatarnikov** [comment]
I would agree, it seems to be an issue with the generation. We should transfer it to `bg-atlasgen`!
